packages/react-components/src/templates/ProfileHeader.tsx

```diff
@@ -102,9 +104,9 @@ const ProfileHeader: React.FC<ProfileProps> = ({
       </section>
       <section css={actionsStyles}>
         <div css={contactStyles}>
-          <Button small primary>
+          <Link small buttonStyle primary href={`mailto:${email}`}>
```

Comment:
Does the email need to be URL encoded? Asking, not sure myself

Reply:
Looked up, `?=&` need to be escaped apparently, and email addresses can in theory contain those, so yeah we probably should. Can make a helper that builds a `mailto` link or so